draft-ietf-httpbis-encryption-encoding.md

```diff
@@ -359,13 +359,13 @@ header field.
 
 aesgcm:
 
-: The "aesgcm" parameter contains the URL-safe base64 [RFC4648] octets of the
-input keying material.
+: The "aesgcm" parameter contains the URL and filename safe base64 [RFC4648]
+  octets of the input keying material with padding removed.
```

Comment:
Well, it turns out that '=' is an invalid character unless you quote the string.  I think that we should be strict here.  I believe that Firefox is, but I could be wrong about that.